docs/src/main/sphinx/sql/drop-function.md

```diff
@@ -0,0 +1,42 @@
+# DROP FUNCTION
+
+## Synopsis
+
+```text
+DROP FUNCTION [ IF EXISTS ] routine_name
+```
+
+## Description
+
+Removes a [](routine-catalog). The value of `routine_name`
+must be fully qualified with catalog and schema location of the routine, unless
+the [default SQL routine storage catalog and
+schema](/admin/properties-sql-environment) are configured.
+
+The optional `IF EXISTS` clause causes the error to be suppressed if
+the function does not exist.
+
+## Examples
```

Comment:
Add an example of dropping a function with a parameter, so that the data type is used
```
DROP FUNCTION multiply_by_two(bigint);

Reply:
Added